src/main/app/src/app/shared/material-form-builder/form-components/message/message-form-field-builder.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import {MessageFormField} from './message-form-field';
+import {MessageLevel} from './message-level.enum';
+import {TextIcon} from '../../../edit/text-icon';
+import {Conditionals} from '../../../edit/conditional-fn';
+import {AbstractFormFieldBuilder} from '../../model/abstract-form-field-builder';
+
+export class MessageFormFieldBuilder extends AbstractFormFieldBuilder {
+
+    public readonly formField: MessageFormField;
+
+    private messageLevel: MessageLevel = MessageLevel.INFO;
+
+    constructor(value: boolean = true) {
```

Comment:
Hernoem value naar initControlValue

Reply:
Ook hier weer, dat heet in alle builders gewoon value. ;-)